### PR TITLE
gravatar urls to https

### PR DESCRIPTION
**Pull request by bhubbard**
I decided to break up my original pull request into smaller bite size requests. Sets all gravatar urls/resources to always be https.

### CODE REVIEW

File: modules/widgets/gravatar-profile.css [hunk 1]
```diff
@@ -40,7 +40,7 @@ only screen and (-o-min-device-pixel-ratio: 3/2),
 only screen and (min--moz-device-pixel-ratio: 1.5),
 only screen and (min-device-pixel-ratio: 1.5) {
 		.widget-grofile .grofile-accounts-logo {
-			background-image: url('//0.gravatar.com/images/grav-share-sprite-2x.png');
+			background-image: url('https://0.gravatar.com/images/grav-share-sprite-2x.png');
```
Comment (jeherve) [inline modules/widgets/gravatar-profile.css]: It might be best to use `https://secure.gravatar.com/images/grav-share-sprite-2x.png` here.

**Comment by bhubbard** [comment]
@jeherve Updated.

File: modules/widgets/gravatar-profile.css [hunk 2]
```diff
@@ -16,7 +16,7 @@
 			content: "" !important; /* Kubrick :( */
 		}
 		.widget-grofile .grofile-accounts-logo {
-			background-image: url('//0.gravatar.com/images/grav-share-sprite.png');
+			background-image: url('https://0.gravatar.com/images/grav-share-sprite.png');
```
Comment (jeherve) [inline modules/widgets/gravatar-profile.css]: You'll need to do the same here.

**Comment by bhubbard** [comment]
@jeherve Done.

**Pull request merged**